src/app/components/layout/navbar/navbar.component.html

```diff
@@ -67,14 +67,14 @@
                 </tr>
               </table>
             </div>
-            <hr class="dropdown-divider">
+            <hr class="dropdown-divider" *ngIf="servicesRecord.length !== 0" >
```

Comment:
do we need this here as well?

Reply:
yes, because if there is no service list below <hr>, then there is no sense to show <hr>